### DISCUSSION TITLE
How to pull application logs

### DISCUSSION THREAD

**Discussion by jmaddr** [teslamate-org/teslamate#1087] [Q&A]
I'm having an issue signing in where I get "Error: unknown" in red at the top at initial sign-in on a completely new docker install.  I looked around but could not find how to pull application logs (container logs don't give anything useful). I think it might be a DNS issue.  

Any help?  TIA.

**Comment by cwanja** [comment]
To get the TeslaMate Docker container logs, for example, use the following command: `docker-compose logs teslamate`.

**Reply by SiNaPsEr0x** [reply]
thanks for command

**Comment by jmaddr** [comment]
Thanks @cwanja.  I did look at the container logs but they didn't result in any useful information.  I've even tailed it while recreating the issue.  I was hoping there might be a shell script I could execute through a docker exec command.  I'm getting the following error and I'm having trouble tracking it down.

<img width="340" alt="image" src="https://user-images.githubusercontent.com/15809711/99529960-a8332e80-296e-11eb-9f7b-de94e676a83d.png">

It's a very generic error and the exact same one you get if your credentials are wrong (which mine aren't).  I'll likely open another question on this since my title is misleading trying to track it here.

**Comment by emilbayes** [comment]
Hey, just had a similar issue. You know it's a DNS issue (out to the world) if you see `:nxdomain` from the `teslamate` container, or if you see `nc ...` when running `docker-container top` (DNS between containers)
